### PR TITLE
tbDEX v1 Release

### PR DESCRIPTION
**Pull request by leordev**
Release checklist: https://github.com/TBD54566975/developer.tbd.website/issues/1362

Release process is complete. This PR is already pointing to the newly 1.0.0 versions of all tbDEX packages!

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206976370475216

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,9 +35,9 @@
     "@tbdex/protocol": "0.28.0",
     "@web5/api": "0.8.4",
     "@web5/credentials": "0.4.5",
-    "@web5/crypto": "0.4.1",
-    "@web5/crypto-aws-kms": "0.2.1",
-    "@web5/dids": "0.4.3",
+    "@web5/crypto": "0.4.0",
+    "@web5/crypto-aws-kms": "0.2.0",
```
Comment (angiejones) [inline package.json:39]: this should be '0.2.1' right?

File: package.json [hunk 2]
```diff
@@ -35,9 +35,9 @@
...
+    "@web5/dids": "0.4.1",
```
Comment (angiejones) [inline package.json:40]: this should be 0.4.3

File: package.json [hunk 3]
```diff
@@ -35,9 +35,9 @@
     "@tbdex/protocol": "0.28.0",
     "@web5/api": "0.8.4",
     "@web5/credentials": "0.4.5",
-    "@web5/crypto": "0.4.1",
-    "@web5/crypto-aws-kms": "0.2.1",
-    "@web5/dids": "0.4.3",
+    "@web5/crypto": "0.4.0",
```
Comment (angiejones) [inline package.json:38]: this should be 0.4.1

**Review by angiejones — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (angiejones) [reply package.json:38]:
```suggestion
    "@web5/crypto": "0.4.1",
```

File: package.json (hunk 1 above)
Comment (angiejones) [reply package.json:39]:
```suggestion
    "@web5/crypto-aws-kms": "0.2.1",
```

File: package.json (hunk 2 above)
Comment (angiejones) [reply package.json:40]:
```suggestion
    "@web5/dids": "0.4.3",
```

**Review by angiejones — APPROVED** [review]

**Comment by ALRubinger** [comment]
@leordev Kotlin SDKs are released to Maven Central as `1.0.0` for all. Verified in this PR:

```
developer.tbd.website/site/testsuites/testsuite-kotlin $> mvn dependency:tree -Dincludes=xyz.block
[INFO] --- dependency:3.6.1:tree (default-cli) @ kotlin-testsuite ---
[INFO] website.tbd.developer.site:kotlin-testsuite:jar:0.1.0-SNAPSHOT
[INFO] \- xyz.block:tbdex:pom:1.0.0:compile
[INFO]    +- xyz.block:tbdex-httpclient:jar:1.0.0:compile
[INFO]    +- xyz.block:tbdex-httpserver:jar:1.0.0:compile
[INFO]    \- xyz.block:tbdex-protocol:jar:1.0.0:compile
[INFO]       \- xyz.block:web5:pom:1.0.0:compile
[INFO]          +- xyz.block:web5-common:jar:1.0.0:compile
[INFO]          +- xyz.block:web5-credentials:jar:1.0.0:compile
[INFO]          +- xyz.block:web5-crypto:jar:1.0.0:compile
[INFO]          +- xyz.block:web5-dids:jar:1.0.0:compile
[INFO]          \- xyz.block:web5-jose:jar:1.0.0:compile
```

**Pull request merged**